README.md

```diff
@@ -1,4 +1,4 @@
-# Teal: Interactive Exploratory Data Analysis with Shiny Web-Applications <a href='https://insightsengineering.github.io/teal/'><img src="man/figures/teal.png" align="right" height="139" style="max-width: 100%;"/></a  >
+# teal: Interactive Exploratory Data Analysis with Shiny Web-Applications <a href='https://insightsengineering.github.io/teal/'><img src="man/figures/teal.png" align="right" height="139" style="max-width: 100%;"/></a  >
```

Comment:
What do you think about removing image and keeping duck as icon :D

```suggestion
# teal 🦆: Interactive Exploratory Data Analysis with Shiny Web-Applications  
```

Reply:
I am good with emoji icon but would like to keep the hexagon badge also